packages/adminrouter/extra/src/Makefile.common

```diff
@@ -87,18 +74,18 @@ update-devkit: clean-devkit-container
 			/bin/bash -c "\$$OPENRESTY_COMPILE_SCRIPT" && \
 	docker commit $$(docker ps -a -q -f name=$(DEVKIT_NAME)) \
 		mesosphere/$(DEVKIT_NAME):full
-	@docker rm $(DEVKIT_NAME)
+	@docker rm -f $(DEVKIT_NAME)
```

Comment:
What do you think of making this `$(MAKE) clean-devkit-container`? To me this would be more clear but also allow us to add changes in just one place.

Saying that - why do we need this at all?

If we do need it:

> This, in turn, stems from the fact that the `update-devkit` target does not call `clean-containers` target just like AR CI does[3]. I will adjust the PR in order to prevent this scenario from happening again.

from JIRA tells me that the problem wasn't that `-f` didn't exist, so maybe this change isn't needed?

Reply:
> What do you think of making this $(MAKE) clean-devkit-container? To me, this would be more clear but also allow us to add changes in just one place.

I am OK with that if you insist. For me, the mental extra effort of going through another layer of abstraction (i.e. calling different makefile target) is not worth avoiding coping just one line of code. Let me know what suits you best.

> Saying that - why do we need this at all?
I would like the Makefile to clean after itself and not to leave artefacts lying around, it does not seem right. We cannot launch `docker run` (see 3 lines earlier) with `--rm` as we need to commit the container, so the idea is that we clean the container up right after it has been committed to the new image.

> ...from JIRA tells me that the problem wasn't that -f didn't exist, so maybe this change isn't needed?

This is just a part of `making it more robust`. Theoretically, I do not think there could be a situation when the container would still be running and thus the `-f` flag would be still required, but OTOH adding this option does not hurt and *may* at some point prevent false negatives. Let me know if you have a strong opinion on that.

Reply:
> I am OK with that if you insist

I don't insist. Changing it is my preference. However, you are the maintainer of this and use it more than me, and it is not a strong preference so I will instead leave it to your preference.

I also don't have a strong opinion on the other points and am happy with you making the judgement call. I also like the Makefile cleaning up after itself. As a reviewer I wanted to separate: "What do we thing is necessary to solve the issue" vs "What is extra safety". This is now clear and I'm happy.

Reply:
> I will instead leave it to your preference.

Thank you, let's please leave it as then.